seqr/views/utils/orm_to_json_utils.py

```diff
@@ -693,11 +697,20 @@ def get_project_collaborators_by_username(project, include_permissions=True):
             collaborator, include_permissions, can_edit=True
         )
 
+    if project_has_anvil(project):
+        anvil_collaborators = get_workspace_collaborators(user, project.workspace_namespace, project.workspace_name)
+        collaborators.update({
+            collab.username: _get_collaborator_json(collab, include_permissions,
+                                                    can_edit = anvil_collaborators[collab.email] == CAN_EDIT,
+                                                    check_anvil = True)
```

Comment:
so it looks like `check_anvil` here will set the `isAnvil ` attribute based on whether or not that user is currently logged in through google. But I think what we actually want here is for `isAnvil` to be True for all of these anvil collaborators. I think it would be clearer if you changed `_get_json_for_user` to take `is_anvil` as the optional argument, instead of `check_anvil`

Reply:
Okay